base/lttagging.dtx

```diff
@@ -621,7 +644,215 @@
+\AssignSocketPlug{tagsupport/math/luamml/mtable/finalizecol}{noop}
+%    \end{macrocode}
+% \end{socketdecl}
+% 
+% 
+% \begin{socketdecl}{tagsupport/math/luamml/mtable/finalize}
+% This sockets is used at the end of alignment environment to finalize the
+% mtable code. It should be used normally with \cs{UseExpandableTaggingSocket}.
+%    \begin{macrocode}
+\NewSocket{tagsupport/math/luamml/mtable/finalize}{1}
+\AssignSocketPlug{tagsupport/math/luamml/mtable/finalize}{noop}
+%    \end{macrocode}
+% \end{socketdecl}
+%
+% \begin{socketdecl}{tagsupport/math/luamml/mtable/aligncol}
+% This sockets is used in multline to add attributes
+% describing the alignment to the left and right.
+% It takes an argument, the alignment.
+%    \begin{macrocode}
+\NewSocket{tagsupport/math/luamml/mtable/aligncol}{1}
+\AssignSocketPlug{tagsupport/math/luamml/mtable/aligncol}{noop}
+%    \end{macrocode}
+% \end{socketdecl}
+
+% 
+% \begin{socketdecl}{tagsupport/math/luamml/mtable/innertable/save}
+% This socket is used in \cs{endaligned} to save the table. It takes no argument.
+%    \begin{macrocode}
+\NewSocket{tagsupport/math/luamml/mtable/innertable/save}{0}
+%    \end{macrocode}
+% \end{socketdecl}
+%
+% \begin{socketdecl}{tagsupport/math/luamml/mtable/smallmatrix/save}
+% This socket is used in \cs{endsmallmatrix} to save the table. It takes no argument.
+% TODO: Check if this socket and the innertable socket can/should be merged into a more
+% generic version.
+%    \begin{macrocode}
+\NewSocket{tagsupport/math/luamml/mtable/smallmatrix/save}{0}
+%    \end{macrocode}
+% \end{socketdecl}
+% 
+% \begin{socketdecl}{tagsupport/math/luamml/mtable/innertable/finalize}
+% This socket is used e.g. in \cs{endsmallmatrix} and \cs{gathered} to finalize the table. 
+% It takes no argument.
+%    \begin{macrocode}
+\NewSocket{tagsupport/math/luamml/mtable/innertable/finalize}{0}
+%    \end{macrocode}
+% \end{socketdecl}
+% 
+% \begin{socketdecl}{tagsupport/math/luamml/mtable/tag/save}
+% This socket is used to save a tag for later use.
+% has been save before.
+%    \begin{macrocode}
+\NewSocket{tagsupport/math/luamml/mtable/tag/save}{0}
+%    \end{macrocode}
+% \end{socketdecl}
+
+% \begin{socketdecl}{tagsupport/math/luamml/mtable/tag/set}
+% This socket should be used when a tag is placed. It inserts a tag that
+% has been save before.
+%    \begin{macrocode}
+\NewSocket{tagsupport/math/luamml/mtable/tag/set}{0}
+%    \end{macrocode}
+% \end{socketdecl}
+% 
+% \paragraph{mbox socket}
+%
+% \begin{socketdecl}{tagsupport/math/luamml/hbox}
+%  This socket is used around \cs{hbox}
+%  inside an \cs{mbox}, \cs{makebox} and \cs{text} and annotates
+%  the content if the \cs{hbox} is used inside math.
+%  The real plug is set by luamml but a default plug is defined
+%  here so that the socket can also be used if luamml is not used.
+%    \begin{macrocode}
+\NewSocket{tagsupport/math/luamml/hbox}{2}
+\NewSocketPlug{tagsupport/math/luamml/hbox}{default}{#2}
```

Comment:
I suggest to use "transparent" as the plug name when it should return the payload but does no tagging

Reply:
done